packages/manager/core/generator/app/templates/src/components/layout-helpers/Listing/dataGrid.tsx

```diff
@@ -48,10 +50,10 @@ export const Datagrid: React.FC<Props> = ({ serviceKey, data }) => {
         </tr>
       </thead>
       <tbody>
-        {data.map((service: any, index: number) => (
+        {data.map((service) => (
           <tr
             className="bg-white border-2 border-solid border-ods-primary-100 background-ods-primary-000"
-            key={`datagrid-tr-${tableHeaders[index]}`}
+            key={`datagrid-tr-${crypto.randomUUID()}`}
```

Comment:
why the need of using a random uid ?

it will also be generated at each render for each line of data which is not great

Reply:
Yes we don't have proper id cause data is dynamic, i am not a big fan of this solution just to not have the sonar warning ...